lib/routes.js

```diff
@@ -513,6 +510,10 @@ Based on Rails RAILS_VERSION routes of APP_CLASS
     define([], function() {
       return result;
     });
+  } else if (typeof module !== "undefined" && module !== null) {
```

Comment:
I've been using js-routes with webpacker since we migrated our javascript from the assest pipeline.

Following the [change](https://github.com/railsware/js-routes/commit/7c654aa0c3055e3cf48a87050fad96a8befd9bad#diff-e04a6967ae1704362c5f4fccb26e7b29296144d2bc5f3c785974d71457418c8aR14) in 7c654aa0c3, I had to specify an initaliser to redefine the namespace (to keep our setup working):

```ruby
JsRoutes.setup do |config|
  config.namespace = "Routes"
end
```

I believe the namespace is no longer honoured since this change, and I'm getting an error of:

![Screenshot 2020-12-18 at 13 01 21](https://user-images.githubusercontent.com/1736807/102617492-30a11c80-4131-11eb-8599-2a90525fb265.png)

Perhaps I'm not importing js-routes in the way that was intended? 

`app/javascript/javascripts/index.coffee.erb`
```coffee
# js-routes
# https://github.com/railsware/js-routes/issues/234#issuecomment-380794815
import Routes from "./routes.js.erb";
# Make the routes available globally
window.Routes = Routes;
```

Where `routes.js.erb` is defined as:

```erb
/* rails-erb-loader-dependencies ../config/routes */
var routes = {};

(function() {
    <%= JsRoutes.generate %>
}.call(routes));

export default routes.Routes;

```

Reply:
Your way of import is pretty odd. I think you just need the following:

```
const routes = <%= JsRoutes.generate %>
export default routes;
```

Also `namespace` option is not needed in this case.

Reply:
Thanks for the information. 

I was getting the same error using the suggestion: 

```
TypeError: Cannot assign to read only property 'exports' of object '#<Object>' error
```

But it seems to work if I used the CommonJS syntax, e.g:

```
const routes = <%= JsRoutes.generate %>;
module.exports = routes;
```